specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Pipeline.json

```diff
@@ -1615,7 +1615,11 @@
       "properties": {
         "columnMappings": {
           "type": "object",
-          "description": "Column mappings. Type: string (or Expression with resultType string)."
+          "description": "Column mappings. Example: \"UserId: MyUserId, Group: MyGroup, Name: MyName\" Type: string (or Expression with resultType string)."

```

Comment:
I meant, x-ms-example, like this:
https://github.com/iamapi/azure-rest-api-specs/tree/a3c2bff7ee233f1de44032b1fccc27450874c77f/specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/examples